<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>informe_tipo_proveedor</class>
 <widget class="QDialog" name="informe_tipo_proveedor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tipo de Proveedor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cuentas:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cuentas_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="solo_activo_checkBox">
       <property name="text">
        <string>&amp; S贸lo Activo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="solo_no_activo_checkBox">
       <property name="text">
        <string>S贸lo No activo</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Tipo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="tipo_proveedor_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Homologaci贸n</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="homologacion_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Actividad</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="actividad_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="acceso_lopd_checkBox">
       <property name="text">
        <string>Acceso LOPD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="acceso_prl_checkBox">
       <property name="text">
        <string>Acceso PRL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="cert_ss_checkBox">
       <property name="text">
        <string>Solicita Certificado S.S.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="no_correo_pub_checkBox">
       <property name="text">
        <string>No correo publicitario</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="cumplimiento_lopd_checkBox">
       <property name="text">
        <string>Contrato LOPD</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="coord_prl_checkBox">
       <property name="text">
        <string>Coordinaci贸n PRL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="cert_hda_checkBox">
       <property name="text">
        <string>Solicita Certificado Hacienda</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="sin_mov_checkBox">
         <property name="text">
          <string>Sin mov. desde</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="fecha_sin_mov_dateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="copiar_pushButton">
       <property name="text">
        <string>&amp;Copiar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimir_pushButton">
       <property name="text">
        <string>&amp;Imprimir horizontal</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/Print-icon.png</normaloff>:/iconos/graf/Print-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprime_ver_pushButton">
       <property name="text">
        <string>Imprimir &amp;vertical</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/Print-icon.png</normaloff>:/iconos/graf/Print-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salir_pushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cuentas_lineEdit</tabstop>
  <tabstop>solo_activo_checkBox</tabstop>
  <tabstop>solo_no_activo_checkBox</tabstop>
  <tabstop>tipo_proveedor_comboBox</tabstop>
  <tabstop>homologacion_comboBox</tabstop>
  <tabstop>actividad_comboBox</tabstop>
  <tabstop>acceso_lopd_checkBox</tabstop>
  <tabstop>acceso_prl_checkBox</tabstop>
  <tabstop>cert_ss_checkBox</tabstop>
  <tabstop>no_correo_pub_checkBox</tabstop>
  <tabstop>cumplimiento_lopd_checkBox</tabstop>
  <tabstop>coord_prl_checkBox</tabstop>
  <tabstop>cert_hda_checkBox</tabstop>
  <tabstop>sin_mov_checkBox</tabstop>
  <tabstop>fecha_sin_mov_dateEdit</tabstop>
  <tabstop>copiar_pushButton</tabstop>
  <tabstop>imprimir_pushButton</tabstop>
  <tabstop>imprime_ver_pushButton</tabstop>
  <tabstop>salir_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salir_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>informe_tipo_proveedor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
